wgpu-types/src/lib.rs

```diff
@@ -809,6 +809,17 @@ pub enum TextureFormat {
     Bc7RgbaUnormSrgb = 51,
 }
 
+impl TextureFormat {
+    pub fn is_depth(self) -> bool {
+        match self {
+            TextureFormat::Depth32Float
```

Comment:
Another thing I forgot to mention on the review (sorry!).
We don't want copies from depth24plus formats. Only Depth32Float is allowed.

Reply:
Ah, don't worry. Thanks for checking.

In that case, I think I will remove this function from wgpu-types since it doesn't seem useful anymore. And instead add `is_valid_copy_destination_texture_format` and `is_valid_copy_source_texture_format` to wgpu-core. Which are again quite long, but since we can't add it to `TextureFormat` inside of wgpu-core it seems like a reasonable choice.